crates/solidity/testing/snapshots/bindings_output/modifiers/simple/generated/0.5.0-success.txt

```diff
@@ -0,0 +1,37 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+References and definitions: 
+    ╭─[input.sol:1:1]
+    │
+  1 │ contract FunctionModifier {
+    │          ────────┬───────  
+    │                  ╰───────── def: 1
+  2 │     bool public locked;
+    │                 ───┬──  
+    │                    ╰──── def: 2
+    │ 
+  4 │     modifier noReentrancy() {
+    │              ──────┬─────  
+    │                    ╰─────── def: 3
+  5 │         require(!locked, "No reentrancy");
+    │         ───┬───  ───┬──  
+    │            ╰───────────── ambiguous: built-in, built-in
```

Comment:
IIUC, we are returning multiple definitions in this case (overloads) .. should we print both instead of `ambiguous` here? it would be a "sucessful" result still..

Reply:
also, I suggest removing the `require()` here, since that is not testing modifiers .. and we can add a separate test for it ..

Reply:
Agreed on both: remove the ambiguous since we're moving away complexity away from the bindings and multiple definitions should be expected more; and it should still be considered successful.